### ISSUE TITLE
CMDPHP PHP NOTICE Backtrace / PHP NOTICE in Plugin 'thold - ERROR

### ISSUE THREAD

**Issue by zuka1337** [Cacti/plugin_thold#436]
Installed new LAB.

**Detected a log ERROR**:

Thold Version: 1.4
Cacti Version 1.2.12

```
2020-06-16 18:11:17 - CMDPHP PHP ERROR NOTICE Backtrace: (/plugins/thold/thold_graph.php[56]:tholds(), /plugins/thold/thold_graph.php[564]:thold_modify_values_by_cdef(), /plugins/thold/thold_functions.php[3688]:thold_build_cdef(), /plugins/thold/thold_functions.php[4035]:CactiErrorHandler())
2020-06-16 18:11:17 - ERROR PHP NOTICE in Plugin 'thold': Use of undefined constant LOG_VERBOSITY_MEDIUM - assumed 'LOG_VERBOSITY_MEDIUM' in file: /var/www/html/cacti/plugins/thold/thold_functions.php on line: 4035
2020-06-16 18:11:17 - CMDPHP PHP ERROR NOTICE Backtrace: (/plugins/thold/thold_graph.php[56]:tholds(), /plugins/thold/thold_graph.php[564]:thold_modify_values_by_cdef(), /plugins/thold/thold_functions.php[3687]:thold_build_cdef(), /plugins/thold/thold_functions.php[4035]:CactiErrorHandler())
```

**The same Error in :**
Thold Version 1.3.2
Cacti Version 1.2.7

```
2020/06/17  10:51:28 - CMDPHP PHP ERROR NOTICE Backtrace:   (/plugins/thold/thold_graph.php[55]:tholds(),  /plugins/thold/thold_graph.php[556]:thold_modify_values_by_cdef(),  /plugins/thold/thold_functions.php[3665]:thold_build_cdef(),  /plugins/thold/thold_functions.php[4012]:CactiErrorHandler())
--
2020/06/17 10:51:28 - ERROR PHP NOTICE in  Plugin 'thold': Use of  undefined constant LOG_VERBOSITY_MEDIUM - assumed 'LOG_VERBOSITY_MEDIUM'  in file: /common/cacti/plugins/thold/thold_functions.php  on line: 4012
2020/06/17 10:51:28 - CMDPHP PHP ERROR NOTICE Backtrace:   (/plugins/thold/thold_graph.php[55]:tholds(),  /plugins/thold/thold_graph.php[556]:thold_modify_values_by_cdef(),  /plugins/thold/thold_functions.php[3664]:thold_build_cdef(),  /plugins/thold/thold_functions.php[4012]:CactiErrorHandler())
```

Can you tell me how can I solve this issue?

**Comment by netniV** [comment]
That particular constant has been around for a long long time so it should be available to you

**Comment by TheWitness** [comment]
It's actually POLLER_VERBOSITY_{}.  Should just relabel them to LOG_LOG ... LOG_DEVDBG though.

**Comment by TheWitness** [comment]
This was already fixed in #433.
